pyproject.toml

```diff
@@ -0,0 +1,110 @@
+[build-system]
+requires = ["setuptools>=64", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "PandABlocks-rootfs"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+description = "Test copier template"
+dependencies = [] # Add project dependencies here, e.g. ["click", "numpy"]
+dynamic = ["version"]
+license.file = "LICENSE"
+readme = "README.md"
+requires-python = ">=3.7"
+
+[project.optional-dependencies]
+dev = [
+    "copier",
+    "mypy",
+    "myst-parser",
+    "pipdeptree",
+    "pre-commit",
+    "pydata-sphinx-theme>=0.12",
+    "pytest",
+    "pytest-cov",
+    "ruff",
+    "sphinx-autobuild",
+    "sphinx-copybutton",
+    "sphinx-design",
+    "tox-direct",
+    "types-mock",
+]
+
+[project.scripts]
+PandABlocks-rootfs = "pandablocks.__main__:main"
+
+[project.urls]
+GitHub = "https://github.com/shihab-dls/PandABlocks-rootfs"
+
+[[project.authors]] # Further authors may be added by duplicating this section
```

Comment:
Project author should be Michael Abbott (and maybe Tom C as well. We should check...)

Reply:
I've changed it to Tom Cobb for now, so I'll resolve after we've checked